pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -1370,6 +1370,12 @@ type KubeletConfig struct {
 	// MemorySwap configures swap memory available to container workloads.
 	// +optional
 	MemorySwap *MemorySwapConfiguration `json:"memorySwap,omitempty" protobuf:"bytes,26,opt,name=memorySwap"`
+	// MaxParallelImagePulls describes the maximum number of image pulls in parallel. The value must be a positive number.
+	// When updating this field, the SerializeImagePulls (pull one image at a time) must be set to false.
+	// Setting it to nil means no limit.
+	// +optional
+	// Default: nil
```

Comment:
Nit: We usually set the optional tag at the end (although I see that in many occurrences it is not set at the end 🙈 ):

```suggestion
	// Default: nil
	// +optional
```